src/create_disk.sh

```diff
@@ -86,6 +87,16 @@ do
 done
 ostree admin deploy "${deploy_ref}" --sysroot rootfs --os "$os_name" $kargsargs
 
+# See the equivalent code in gf-anaconda-cleanup
```

Comment:
This is just an issue on rhcos where we don't run tmpfiles in the initramfs, yes?

Reply:
Yeah...as the commit message says, doing so fails due to SELinux policy.